testsuite/tests/tool-ocamlopt-stop-after/stop_after_scheduling.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+asm=${test_build_directory}/stop_after_scheduling.${asmext}
+obj=${test_build_directory}/stop_after_scheduling.${objext}
+cmx=${test_build_directory}/stop_after_scheduling.cmx
+
+# Check that cmx is generated but asm and obj are not
+if [ -e "$asm" ]; then
+    test_result=${TEST_FAIL};
+else if [ -e "$obj" ]; then
+         test_result=${TEST_FAIL};
+     else if [ -e "$cmx" ]; then
+              test_reslut=${TEST_PASS};
```

Comment:
There is a typo at `test_result` here and below.

Reply:
thank you! Fixed. I found another one in an unrelated tests/unwind/check-linker-version.sh and fixed it too while at it. Is it ok to include this small unrelated change?